roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstanceHelper.java

```diff
@@ -126,6 +126,7 @@ public static Long synchronizeIfUpdated(User user) throws GenericException, NotF
     if (localInstance != null && SynchronizingStatus.ACTIVE.equals(localInstance.getStatus())) {
       Date fromDate = localInstance.getLastSynchronizationDate();
       Date toDate = new Date();
+      total+= SyncUtils.getUpdatesFromDistributedInstance(localInstance);
```

Comment:
Please also change the name of this method "synchronizeIfUpdated" to "retrieveLocalInstanceUpdates"

Reply:
done